.github/CHANGELOG.md

```diff
@@ -116,6 +116,11 @@
   [(#378)](https://github.com/XanaduAI/strawberryfields/pull/378)
   [(#381)](https://github.com/XanaduAI/strawberryfields/pull/381)
 
+* Strawberry Fields can now access the Xanadu Cloud device specifications API.
+  The ``Connection`` class has a new method ``Connection.get_device(target)``,
```

Comment:
(Minor suggestion): having `Connection.get_device(target)` might imply that it is a static method as it takes is no `self` argument
```suggestion
  The ``Connection`` class has a new method ``Connection.get_device``,
```